lib/_http_server.js

```diff
@@ -331,7 +331,7 @@ ServerResponse.prototype._implicitHeader = function _implicitHeader() {
 };
 
 ServerResponse.prototype.writeHead = writeHead;
-function writeHead(statusCode, reason, obj) {
+function writeHead(statusCode, statusMessage, headers) {
```

Comment:
I think the diff would be much smaller and we could avoid the awkward `_headers` if we didn't change the parameter names.

Reply:
I'd just save the parameter/variable name changes for a different PR (or at least a separate commit) to make things clearer.

Reply:
I would bet `headers` wasn't originally used as the parameter name because it was going to be used as a variable in the function's body.

Reply:
all right I can make it into another pr

Reply:
> I would bet `headers` wasn't originally used as the parameter name because it was going to be used as a variable in the function's body.

yep